dev/io.openliberty.microprofile.openapi.4.0.internal_fat/fat/src/io/openliberty/microprofile/openapi40/fat/versionwarning/VersionWarningTest.java

```diff
@@ -94,16 +96,30 @@ public void cleanupApps() throws Exception {
     @SuppressWarnings("unchecked")
     @Test
     public void testVersionWarning() throws Exception {
-        setOpenAPIVersion("3.0");
+        setOpenAPIVersion("3.0", server);
 
-        WebArchive war = ShrinkWrap.create(WebArchive.class, "testWarning.war")
+        WebArchive war = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war")
                                    .addPackage(VersionWarningApplication.class.getPackage());
 
         server.setMarkToEndOfLog();
         ShrinkHelper.exportDropinAppToServer(server, war, SERVER_ONLY);
 
+        //Assert the version warnings appear on app startup
+        assertVersionWarnings("Failed to find warning on app startup");
+
+        //Assert the version warnings appear on app restart
+        server.setMarkToEndOfLog();
+        ApplicationMBean mbean = server.getApplicationMBean(APP_NAME);
+        mbean.stop();
+        mbean.start();
+        server.findStringsInLogsAndTraceUsingMark("CWWKT0016I:.*testWarning");
```

Comment:
1. Should be looking for `CWWKZ0001I` (Application testWarning started in 0.927 seconds.)
2. Should be using `waitForStringsInLogs`
  - Need to wait here to make sure the log is there
  - Don't need to look in the trace

Reply:
Decided we probably need to use `LibertyServer.restartDropinsApplication` instead of the mbean restart until we update the logic of `LibertyServer.waitForAppState` to understand the app updated message as indicating that the app is started.